src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,25 @@
+@use '../utils/variables' as *;
+
+.stars {
+  display: flex;
+  justify-content: flex-start;
+
+  &__star {
+    background-image: url('../images/star.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+
+    width: $stars-size;
```

Comment:
Ensure that the `$stars-size` variable is defined in the `variables` file. If it's not defined, the width and height of the stars might not be set correctly.